youtube_dl/extractor/canvas.py

```diff
@@ -110,10 +104,153 @@ def _real_extract(self, url):
         return {
             'id': video_id,
             'display_id': display_id,
-            'title': title,
-            'description': self._og_search_description(webpage),
             'formats': formats,
             'duration': float_or_none(data.get('duration'), 1000),
             'thumbnail': data.get('posterImageUrl'),
             'subtitles': subtitles,
         }
+
+    def _real_extract(self, url):
+        mobj = re.match(self._VALID_URL, url)
+        site_id, display_id = mobj.group('site_id'), mobj.group('id')
+
+        webpage = self._download_webpage(url, display_id)
+
+        title = (self._search_regex(
+            r'<h1[^>]+class="video__body__header__title"[^>]*>(.+?)</h1>',
+            webpage, 'title', default=None) or self._og_search_title(
+            webpage)).strip()
+
+        description = self._og_search_description(webpage)
+
+        video_id = self._html_search_regex(
+            r'data-video=(["\'])(?P<id>(?:(?!\1).)+)\1', webpage, 'video id',
+            group='id')
+
+        info = self._extract_info(site_id, video_id, display_id)
+        info.update({
+            'title': title,
+            'description': description,
+        })
+        return info
+
+
+class VrtNUIE(CanvasIE):
+    _VALID_URL = r'https?://(?:www\.)?vrt\.be/vrtnu/(?:[^/]+/)*(?P<id>[^/?#&]+)'
+    _TESTS = [
+        {
+            'url': 'https://www.vrt.be/vrtnu/a-z/trapped/1/trapped-s1a4/',
+            'info_dict': {
+                'id': 'md-ast-01e7d193-63ab-4d89-9ab1-96b23315ebec_1489574042173',
+                'ext': 'mp4',
+                'title': 'Trapped',
+                'description': 'md5:6f4f779d176252f887fdd938910d6094',
+                'duration': 3011.17,
+                'thumbnail': r're:^https?://.*\.jpg$'
+            },
+            'skip': 'This video is only available for registered users'
```

Comment:
I cannot reproduce this, I have to authenticate in order to view [the video](https://www.vrt.be/vrtnu/a-z/trapped/1/trapped-s1a4/) in a browser (Firefox, Chromium, Midori).